SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# DHIS2 Security Policy
+
+The DHIS2 security team participates in responsible disclosure and welcomes collaboration with the wider community on security issues.
+To report a security issue, please email security@dhis2.org with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue.
```

Comment:
Lines 3-6 are rendered as a single one-line paragraph. It would be more readable if we added empty lines between each line as maximal clarity should be our goal with this text.

Reply:
👍  done, much more clear!